scripts/lib/file

```diff
@@ -49,15 +49,16 @@ EOF
 # create_operator_group_file creates an operator group for the operator CSV.
 function create_operator_group_file() {
   local op_group_file="$1"
-  local name="$2"
-  local service_account="$3"
+  local op_name="$2"
+  local csv_file="$3"
   local namespace="$4"
 
+  local service_account="$(yq r "$csv_file" "spec.install.spec.deployments[0].spec.template.spec.serviceAccountName")"
   cat <<EOF > "$op_group_file"
 apiVersion: operators.coreos.com/v1alpha2
 kind: OperatorGroup
 metadata:
-  name: ${name}-og
+  name: ${op_name}-og
   namespace: $namespace
```

Comment:
This function should take the operator's supported installModes into account

Reply:
`create_operator_group_file` will only create an `OperatorGroup` if `AllNamespaces` is false. Does this seem reasonable?

Reply:
Yes, that will work. We should add a quick check that `SingleNamespace` is set to true as well. If `AllNamespaces` and `SingleNamespace` are both false, then we should error out early.